UnityProject/Assets/Scripts/UI/Core/PrefabTracker.cs

```diff
@@ -28,6 +28,7 @@ public class PrefabTracker : MonoBehaviour
 		}
 
 		[SerializeField] private string foreverID;
+		[field:SerializeField] public string AlternativePrefabName { get; set; }
```

Comment:
```suggestion
		[field:SerializeField] public string AlternativePrefabName { get; private set; }
```

@MaxIsJoe accept this change if thats easier than opening the branch for you